frontend/src/app/cookie-law-banner/cookie-law-banner.component.css

```diff
@@ -0,0 +1,20 @@
+#cookielaw_banner {
```

Comment:
can we stick with dash instead od undersscore? previously introduced components contains dashes in class names

Reply:
I assumed dashes for class names, underscores for id names

Reply:
I don't know, why do we need to use ids here instead of classes?

Reply:
for the banner, I used an id because it's being retrieved and modified in the `document.getElementById('cookielaw_banner').style.display = shouldHide ? 'none' : 'visible'`